recipes/python-keystoneclient/meta.yaml

```diff
@@ -0,0 +1,75 @@
+
+test:
+  imports:
+    - keystoneclient
+
+about:
+  home: The package home page
+  license: Apache
+  license_family: APACHE
+  license_file: ''
+  summary: Summary of the package
+  description: "========================\nTeam and repository tags\n========================\n\n.. image:: https://governance.openstack.org/badges/python-keystoneclient.svg\n    :target: https://governance.openstack.org/reference/tags/index.html\n\
+    \n.. Change things from this point on\n\nPython bindings to the OpenStack Identity API (Keystone)\n========================================================\n\n.. image:: https://img.shields.io/pypi/v/python-keystoneclient.svg\n\
+    \    :target: https://pypi.python.org/pypi/python-keystoneclient/\n    :alt: Latest Version\n\n.. image:: https://img.shields.io/pypi/dm/python-keystoneclient.svg\n    :target: https://pypi.python.org/pypi/python-keystoneclient/\n\
+    \    :alt: Downloads\n\nThis is a client for the OpenStack Identity API, implemented by the Keystone\nteam; it contains a Python API (the ``keystoneclient`` module) for\nOpenStack's Identity Service.\
+    \ For command line interface support, use\n`OpenStackClient`_.\n\n* `PyPi`_ - package installation\n* `Online Documentation`_\n* `Launchpad project`_ - release management\n* `Blueprints`_ - feature\
+    \ specifications\n* `Bugs`_ - issue tracking\n* `Source`_\n* `Specs`_\n* `How to Contribute`_\n\n.. _PyPi: https://pypi.python.org/pypi/python-keystoneclient\n.. _Online Documentation: https://docs.openstack.org/python-keystoneclient/latest/\n\
+    .. _Launchpad project: https://launchpad.net/python-keystoneclient\n.. _Blueprints: https://blueprints.launchpad.net/python-keystoneclient\n.. _Bugs: https://bugs.launchpad.net/python-keystoneclient\n\
+    .. _Source: https://git.openstack.org/cgit/openstack/python-keystoneclient\n.. _OpenStackClient: https://pypi.python.org/pypi/python-openstackclient\n.. _How to Contribute: https://docs.openstack.org/infra/manual/developers.html\n\
+    .. _Specs: https://specs.openstack.org/openstack/keystone-specs/\n\n.. contents:: Contents:\n   :local:\n\nPython API\n----------\n\nBy way of a quick-start::\n\n    >>> from keystoneauth1.identity\
+    \ import v3\n    >>> from keystoneauth1 import session\n    >>> from keystoneclient.v3 import client\n    >>> auth = v3.Password(auth_url=\"http://example.com:5000/v3\", username=\"admin\",\n    ...\
+    \                     password=\"password\", project_name=\"admin\",\n    ...                     user_domain_id=\"default\", project_domain_id=\"default\")\n    >>> sess = session.Session(auth=auth)\n\
+    \    >>> keystone = client.Client(session=sess)\n    >>> keystone.projects.list()\n        [...]\n    >>> project = keystone.projects.create(name=\"test\", description=\"My new Project!\", domain=\"\
+    default\", enabled=True)\n    >>> project.delete()\n\n\n\n"
+  doc_url: ''
+  dev_url: ''
+
+extra:
+  recipe-maintainers:
+    - jtratner
+    - pkaleta
```

Comment:
@pkaleta -- are you ok being added as a co-maintainer of the `python-keystoneclient` recipe on conda-forge?

http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role

Reply:
sgtm